src/group.js

```diff
@@ -231,9 +231,20 @@ var _ = Mavo.Group = $.Class({
 		this.propagate(obj => {
 			var propertyData = data[obj.property];
 
-			if (obj.alias && data[obj.alias] !== undefined) {
-				copy = copy || $.extend({}, data);
-				propertyData = data[obj.alias];
+			// find first alias with data, load that data, and set to be copied
+			if (obj.alias) {
+				aliasesArr = obj.alias.split(" ");
```

Comment:
I'd be inclined to split where `this.alias` is set so that we don't have to do it on every render. But this is fine too!